test/ApplicationSpec.scala

```diff
@@ -1,15 +1,17 @@
+import org.junit.runner._
 import org.specs2.mutable._
 import org.specs2.runner._
-import org.junit.runner._
-
-import play.api.test._
 import play.api.test.Helpers._
+import play.api.test._
+import testUtils.PlayUtils.fakeApplicationWithGlobal
 
 @RunWith(classOf[JUnitRunner])
 class ApplicationSpec extends Specification {
 
+
   "Application" should {
-    "render the index page" in new WithApplication {
+    
+    "render the index page" in new WithApplication(app = fakeApplicationWithGlobal) {
```

Comment:
I'm not sure, does this test give value? Should we just delete it? Or will it capture something we wouldn't have noticed?